staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3324,6 +3324,10 @@ type PodCondition struct {
 	// Human-readable message indicating details about last transition.
 	// +optional
 	Message string `json:"message,omitempty" protobuf:"bytes,6,opt,name=message"`
+	// The generation of the pod observed at the point this condition is being reported.
```

Comment:
because this is a new field which could be unset (when talking to older API servers), it's worth documenting this as a conditional, e.g.:

```
If set, this represents the .metadata.generation that the condition was set based upon.
```

and if we're going to clear this field until PodObservedGenerationTracking is enabled, this should also document this is an alpha-level field that requires the PodObservedGenerationTracking feature to be enabled, e.g.:

```
This is an alpha field. Enable PodObservedGenerationTracking to be able to use this field.
```

Reply:
Thanks! Done